### PR TITLE
cli: implement mv command

### LINKED ISSUE #97: cli: implement `mv` functionality
```console
$ reana-client mv --help
Usage: reana-client mv [OPTIONS] SOURCE TARGET

  Move files within workspace.

  The ``mv`` command allow to move the files within workspace.

  Examples:

       $ reana-client mv data/input.txt input/input.txt

Options:
  -w, --workflow TEXT      Name or UUID of the workflow. Overrides value of
                           REANA_WORKON environment variable.
  -t, --access-token TEXT  Access token of the current user.
  --help                   Show this message and exit.
```

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #97

### CODE REVIEW

File: cmd/mv.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	sourceExists := false
+	for _, file := range files.Items {
+		if strings.Contains(file.Name, o.source) {
+			sourceExists = true
+			break
+		}
+	}
+	if !sourceExists {
+		return fmt.Errorf("source file(s) %s does not exist in %s workspace", o.source, o.workflow)
+	}
```
Comment (BrunoRosendo) [inline cmd/mv.go:91]: Should we keep this validation? I feel like we could just use the validation from the server but I wasn't sure so I ported the way it's done in Python

Comment (mdonadoni) [reply cmd/mv.go:91]: Yes, I think it is a good idea to remove both of them, as they are already being done server side. However, the server side check that a workflow is not running is not working correctly, so I will open an issue about that.

Comment (mdonadoni) [reply cmd/mv.go:91]: Related issue: https://github.com/reanahub/reana-workflow-controller/issues/469

Comment (BrunoRosendo) [reply cmd/mv.go]: It's done, I had to update the openapi and regenerate the client

**Review by mdonadoni — APPROVED** [review]

File: cmd/mv.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	"reanahub/reana-client-go/client/operations"
+	"reanahub/reana-client-go/pkg/displayer"
+
+	"github.com/spf13/cobra"
+)
+
+const mvDesc = `
+Move files within workspace.
+
+The ` + "``close``" + ` command allows to move the files within workspace.
```
Comment (mdonadoni) [inline cmd/mv.go:23]:
```suggestion
The ` + "``mv``" + ` command allows to move the files within workspace.
```

**Pull request merged**